packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -855,7 +849,7 @@ export class GarbageCollector implements IGarbageCollector {
 		trackState: boolean,
 		telemetryContext?: ITelemetryContext,
 	): ISummarizeResult | undefined {
-		if (!this.configs.shouldRunGC || this.gcDataFromLastRun === undefined) {
+		if (!this.shouldRunGC || this.gcDataFromLastRun === undefined) {
```

Comment:
This is fine (more future-proof) but I think we only need to check `this.gcDataFromLastRun`, right?

Reply:
Yes, you are right.